_posts/2020-10-13-bigsur-new.md

```diff
@@ -0,0 +1,330 @@
+
+So for the past 15 years, macOS has been using the Prelinked Kernel as a form of Kernel and Kext caching. And with macOS Big Sur's new Read-only, snapshot based system volume, a new version of caching has be developed: KernelCollections!
+
+How this differs to previous OSes:
+
+* Kexts can no longer be hot-loaded, instead requiring a reboot to load with `kmutil`
+* The Secure Boot and standard kernel are now one(ie. no more dedicated Immutable Kernel)
+* Symbols are no longer required
+
+The last point is the most important part, as this is what we use for kext injection in OpenCore. Currently Apple has left symbols in place seemingly for debugging purposes however this is a bit worrying as Apple could outright remove symbols in later versions of macOS. But for macOS 11, Big Sur's cycle, we'll be good on that end however we'll be keeping an eye on macOS 12.
+
+## New Kernel Requirements
+
+With this update, the `AvoidRuntimeDefrag` Booter quirk in OpenCore broke. Because of this, the macOS kernel will fall flat when trying to boot. Reason for this is due to `cpu_count_enabled_logical_processors` requiring the MADT (APIC) table, and so OpenCore will now ensure this table is made accessible to the kernel. Users will however need a build of OpenCore 0.6.0 with commit [`bb12f5f`](https://github.com/acidanthera/OpenCorePkg/commit/9f59339e7eb8c213e84551df0fdbf9905cd98ca4) **or newer** to resolve this issue.
+
+To check your OpenCore version, run the following in terminal:
+
+```sh
+nvram 4D1FDA02-38C7-4A6A-9CC6-4BCCA8B30102:opencore-version
+```
+
+If you're not up-to-date, see here on how to upgrade OpenCore: [Updating OpenCore, Kexts and macOS](https://dortania.github.io/OpenCore-Post-Install/universal/update.html)
+
+## Broken Kexts in Big Sur
+
+Unfortunately with the aforementioned KernelCollections, some kexts have unfortunately broken or have been hindered in some way. The main kexts that currently have issues are anything relying on Lilu's userspace patching functionality:
+
+* [WhateverGreen's](https://github.com/acidanthera/WhateverGreen) [DRM](https://dortania.github.io/OpenCore-Post-Install/universal/drm.html) and `-cdfon` patches
+  * Most of WhateverGreen's functions are still working as they're kernelspace based
+* [MacProMemoryNotificationDisabler](https://github.com/IOIIIO/MacProMemoryNotificationDisabler)
+* [SystemProfilerMemoryFixup](https://github.com/Goldfish64/SystemProfilerMemoryFixup)
+
+Thankfully most important kexts rely on kernelspace patcher which is now in fact working again.
+
+## MSI Navi installer Bug Resolved
+
+For those receiving boot failures in the installer due to having an MSI Navi GPU installed, macOS Big Sur has finally resolved this issue!
+
+## New AMD OS X Kernel Patches
+
+For those running on AMD-Based CPUs, you'll want to also update your kernel patches as well since patches have been rewritten for macOS Big Sur support:
+
+* [AMD OSX Kernel Patches](https://github.com/AMD-OSX/AMD_Vanilla)
```

Comment:
```suggestion
* [AMD OSX Kernel Patches](https://github.com/AMD-OSX/AMD_Vanilla/tree/experimental-opencore)
```

Reply:
This PR is to be merged with BigSur's release, and AMD OSX will be merging into mainline as well

Reply:
However keep an eye on the patches